dragula.js

```diff
@@ -419,12 +419,22 @@ function dragula (initialContainers, options) {
     if (drake.dragging) { classes.add(el, 'gu-hide'); }
   }
 
+  function cloneElement (el) {
+    var selector = 'select';  // more types can be added later
```

Comment:
The checked state of radio buttons is also affected. It's because `el.cloneNode(true)` copies that checked state, and because they share the same `name` attribute, only one can be checked. Since the cloned element was created last, that one is checked, and whichever was last checked is unchecked.

For the loop below, adding the following works for radio buttons

```js
    if (cloneDescendants[i].checked) {
      // Change the name of the radio field so that checking the
      // original element again won't uncheck the mirrored element.
      cloneDescendants[i].setAttribute('name', cloneDescendants[i].getAttribute('name') + '_');
      element.checked = true;
    }
```